janitor/functions.py

```diff
@@ -665,10 +628,9 @@ def encode_categorical(
         df = jn.encode_categorical(
                     df,
                     col1 = (categories, order),
-                    col2 = jn.As_Categorical(
-                                categories = [values],
-                                order="sort"/"appearance"/None
-                                )
+                    col2 = (categories = [values],
+                            order="sort"/"appearance"/None
+                           )
```

Comment:
I think this would be misleading for readers of the code. The alternative values should be placed as comments after. Here's an example:

```suggestion
                    col2 = (categories = [values],
                            order="sort"  # or "appearance" or None
                           )
```